tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
+// Included from bootstrap-sass.config.js
```

Comment:
I'll have to see this in the generated samples. Not quite following. Key is that we include the sass helper early, and the easiest way to do that is to put a link to the file in the boostrap variables customization file which gets loaded early.